extensions/wikia/AdEngine/js/video/vastUrlBuilder.js

```diff
@@ -56,21 +56,30 @@ define('ext.wikia.adEngine.video.vastUrlBuilder', [
 		slotParams = slotParams || {};
 
 		var correlator = Math.round(Math.random() * 10000000000),
-			params = [
-				'output=vast',
-				'env=vp',
-				'gdfp_req=1',
-				'impl=s',
-				'unviewed_position_start=1',
-				'iu=' + buildAdUnit(slotParams),
-				'sz=' + getSizeByAspectRatio(aspectRatio),
-				'url=' + encodeURIComponent(loc.href),
-				'description_url=' + encodeURIComponent(loc.href),
-				'correlator=' + correlator,
-				'cust_params=' + getCustomParameters(slotParams)
-			],
+			useMegaAdUnitBuilder = options.useMegaAdUnitBuilder,
+			params,
 			url;
 
+		// fallback to adContext flag
+		if (useMegaAdUnitBuilder === undefined) {
```

Comment:
It doesn't change anything in this case but it's a good practice to not forget about it when using on variables that can be not defined.